packages/manager/apps/hpc-vmware-vsphere/src/routes/routes.constant.ts

```diff
@@ -4,4 +4,9 @@ export const urls = {
   listing: '',
   dashboard: ':serviceName',
   logs: 'logs',
+  datacenters: 'datacenters',
+  users: 'users',
+  security: 'security',
+  operations: 'operation',
```

Comment:
```suggestion
  operations: 'operations',
```

Why it's only one (except security) to be in singular ?

Reply:
https://www.ovh.com/manager/#/dedicated/dedicated_cloud/pcc-51-210-115-127/operation

Following legacy pattern for now